src/ios/TwilioVoicePlugin.m

```diff
@@ -161,11 +161,22 @@ - (void) initializeWithAccessToken:(CDVInvokedUrlCommand*)command  {
 
 - (void) call:(CDVInvokedUrlCommand*)command {
     if ([command.arguments count] > 0) {
+        
+        
+        
         if (command.arguments[0] != [NSNull null]) {
             self.accessToken = command.arguments[0];
         }
         if ([command.arguments count] > 1) {
-            self.outgoingCallParams = command.arguments[1];
+            if ( [allTrim( command.arguments[1] ) length] == 0 ) {
+                NSDictionary *inventory = @{
```

Comment:
this is to handle a string for the number rather than an object right?

Can you add a comment accordingly?

Also improve the variable name (perhaps callParams)